src/components/dialogs/SettingsDialog.vue

```diff
@@ -37,16 +37,22 @@ export default {
       settings: {
         networking: {
           updateInterval: this.getUpdateInterval() / 1_000
+        },
+        appearance: {
+          darkMode: this.getDarkMode()
```

Comment:
Is this just a toggle right now?

Reply:
Can you explain what do you mean by this? It is a boolean/toggle option. 

I think there is probably a better way to do this, but not sure right now what it is.

Reply:
I was just thinking it would be the name of a theme or something, but this is fine for now.